src/java/org/apache/cassandra/service/snapshot/SnapshotDetailsTabularData.java

```diff
@@ -74,14 +81,14 @@ public static void from(TableSnapshot details, TabularDataSupport result, Set<St
     {
         try
         {
-            final String totalSize = FileUtils.stringifyFileSize(details.computeSizeOnDiskBytes());
```

Comment:
not sure why we are stressing so much that this has to `final` ... doesnt matter at all.